packages/specification/schemas/fragments/codes.yaml

```diff
@@ -0,0 +1,116 @@
+MedicationRequestCategory:
```

Comment:
Really think we should get rid of this as we cannot reliably provide the data needed to support it. See below.

Reply:
Removed this along with references.